src/afl-forkserver.c

```diff
@@ -290,21 +291,25 @@ void afl_fsrv_start(afl_forkserver_t *fsrv, char **argv) {
 
   /* Wait for the fork server to come up, but don't wait too long. */
 
-  if (fsrv->exec_tmout) {
```

Comment:
Please add this again, see my other comment 🙃
Might be a handy feature for some.